conda/recipes/cuXfilter/build.sh

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+
+# This assumes the script is executed from the root of the repo directory
+./build.sh cudf
```

Comment:
Should build cuxfilter and cudatashader here. Also folder name to all lowercase.